posts/Information-as-an-Asset-Through-Business-Intelligence-and-Data-Analytics.md

```diff
@@ -0,0 +1,27 @@
+---
+authors:
+- Rita Barlow
+tags:
+- Data Analytics
+- Business Intelligence
+- Data 
+- Big Data 
+- Information 
+date: 2019-02-05T12:21:50.000Z
+title: "Information as an Asset Through Business Intelligence and Data Analytics"
+image: BI_VS_Data_Analytics.png
+---
+## Make Better Decisions Faster
+
+Business Intelligence and Data Analytics are critical practices to ensure your organization is running efficiently and proactively. In order to determine _how_ these fundamentals of data practices can help improve operations within your organization, we must first be able to determine the difference between the two, and answer how they can be utilized moving forward.
+
+**Business Intelligence (BI)** refers to the technologies, applications, and practices used to collect, integrate, analyze, and visualize your organization&#39;s data as valuable information.  This technology-driven process is used to help executives, managers, and other data consumers in your organization make informed business decisions. BI allows your organization to identify business opportunities, gauge productivity, and optimize spending for areas like sales and marketing by predicting customer behavior and analyzing trends over time.
+
+**Data Analytics (DA)** refers to the act of converting unstructured or raw data into a comprehensive or informational format that can be used to support your organization at all levels. Your organization will be able to make quality decisions, improve planning and forecasting, enable consistent data across your enterprise, improve speed, accessibility, ease of use, and ultimately lower costs and increase operational efficiency. The benefits to how Data Analytics can improve your organization are nearly endless. The following are a couple examples where Data Analytics can be applied to your data to make it a powerful informational asset.
+
+- **Customer Experience:** Data Analytics allows you to get to know your customer by compiling and understanding data regarding their habits, preferences, and needs. This enables your organization to tailor offerings and solutions that will optimize your customers experience and outcomes, making them more likely to return to your organization in the future. This could be applied to product offerings, services, geographical locations of stores, frequency of contact, medium of contact, etc.
```

Comment:
```suggestion
- **Customer Experience:** Data Analytics allows you to get to know your customers by compiling and understanding data regarding their habits, preferences, and needs. This enables your organization to tailor offerings and solutions that will optimize your customers' experience and outcomes, making them more likely to return to your organization in the future. This could be applied to product offerings, services, geographical locations of stores, frequency of contact, medium of contact, etc.
```